ui.apps/src/main/content/jcr_root/apps/core/cif/clientlibs/react-components/.content.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
-          jcr:primaryType="cq:ClientLibraryFolder"
-          allowProxy="{Boolean}true"
-          categories="[core.cif.components.react]"/>
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[core.cif.components.react]"
+    jsProcessor="[default:none,min:none]"/>
```

Comment:
Should we disable the AEM minification for the following clientlibs as well? Those are also built using webpack and will automatically be minified during a release.

* /apps/core/cif/clientlibs/common
* /apps/core/cif/components/commerce/product/v1/product
* /apps/core/cif/components/commerce/productcarousel/v1/productcarousel/clientlibs
* /apps/core/cif/components/commerce/productlist/v1/productlist/clientlibs
* /apps/core/cif/components/commerce/searchbar/v1/searchbar/clientlibs
* /apps/core/cif/components/commerce/productteaser/v1/productteaser/clientlibs
* /apps/core/cif/components/structure/navigation/v1/navigation/clientlibs

Reply:
Good point, I'll check this as well and make sure everything works.

Reply:
@herzog31 I disabled the AEM minification for all clientlibs built by webpack and tested again.